### PR TITLE
weekly news

### PR DESCRIPTION
**Pull request by jnoller**
BOOYAH :boom: :cake: :astonished:

### CODE REVIEW

File: _posts/2016-02-11-debugging-docker-affinity-mumble-and-feedback.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+<img class="right" src="{% asset_path weekly-news/mumble.png %}" alt="mumble"/>
+
+Next up; Mumble is a low-latency, multiplatform voice chat software. I remember using it mostly for... well World of Warcraft (don't ask) and Counter Strike but Zack Shoylev shows us how to use it in Docker making it easy to run Murmur in a portable way.
+
+If you like talking with people (it's a life-choice) then check it out - mumble is encrypted voice communications, and if you mesh it with magically renewing TLS certificates via [NGINX with Let's Encrypt]({{ site.baseurl }}/docs/tutorials/nginx-with-lets-encrypt/) it should get you +10 points to house tinfoil! <img class="left" src="{% asset_path weekly-news/tinfoil.jp %}" alt="tinfoil"/>
+
+[Use Mumble on Carina]({{ site.baseurl }}/tutorials/2016-02-05-mumble-on-carina/)
+
+## Publish an ASP.NET website to Carina from the command line
```
Comment (carolynvs) [inline _posts/2016-02-11-debugging-docker-affinity-mumble-and-feedback.md]: We can remove this. It made it into last week's roundup.

File: _posts/2016-02-11-debugging-docker-affinity-mumble-and-feedback.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Check it out: [Running a container using a custom image]({{ site.baseurl }}/troubleshooting/run-container-using-custom-image/)
+
+## Use Mumble on Carina
+
+<img class="right" src="{% asset_path weekly-news/mumble.png %}" alt="mumble"/>
+
+Next up; Mumble is a low-latency, multiplatform voice chat software. I remember using it mostly for... well World of Warcraft (don't ask) and Counter Strike but Zack Shoylev shows us how to use it in Docker making it easy to run Murmur in a portable way.
+
+If you like talking with people (it's a life-choice) then check it out - mumble is encrypted voice communications, and if you mesh it with magically renewing TLS certificates via [NGINX with Let's Encrypt]({{ site.baseurl }}/docs/tutorials/nginx-with-lets-encrypt/) it should get you +10 points to house tinfoil! <img class="left" src="{% asset_path weekly-news/tinfoil.jp %}" alt="tinfoil"/>
```
Comment (rgbkrk) [inline _posts/2016-02-11-debugging-docker-affinity-mumble-and-feedback.md]: tinfoil.jp -> tinfoil.jpg

Comment (jnoller) [inline _posts/2016-02-11-debugging-docker-affinity-mumble-and-feedback.md]: HAHAHA I FIXED IT :cake:

File: _posts/2016-02-11-debugging-docker-affinity-mumble-and-feedback.md (hunk 1 above)
Comment (jnoller) [inline _posts/2016-02-11-debugging-docker-affinity-mumble-and-feedback.md]: I don't see it in my version ?

File: _posts/2016-02-11-debugging-docker-affinity-mumble-and-feedback.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  In this week's roundup:
+categories:
+ - Docker
+ - Affinity
+ - Carina
+ - News
+authorIsRacker: true
+---
+
+Let's encrypt ASP.NET and IRC. Oops, that came out wrong. We've got some excellent new tutorials on using NGINX with Let's Encrypt, getting started with ASP.NET with Visual Studio, publishing an ASP.NET website from the command line, and using Quassel (distributed IRC client). Also news from the world of Docker.
```
Comment (carolynvs) [inline _posts/2016-02-11-debugging-docker-affinity-mumble-and-feedback.md]: Just a wee bit more dangling asp.net right here that can be nuked. :smile:

File: _posts/2016-02-11-debugging-docker-affinity-mumble-and-feedback.md (hunk 1 above)
Comment (carolynvs) [inline _posts/2016-02-11-debugging-docker-affinity-mumble-and-feedback.md]: Magic! I don't see it either anymore.

**Comment by jnoller** [comment]
@carolynvs @rgbkrk please re-check

**Comment by carolynvs** [comment]
:+1: moar pugs

**Pull request merged**